src/redux/thunk.js

```diff
@@ -134,8 +134,8 @@ const addCartItemThunk = item => {
 
 const deleteCartItemThunk = item => {
   return async dispatch => {
-    const response = (await axios.delete(`/api/lineItems/${item.id}`)).data;
-    dispatch(deleteCartItem(response));
+    await axios.delete(`/api/lineItems/${item.id}`);
```

Comment:
will this work for multiple line items?

Reply:
Yes it will work for all items
for multiple items increase or decrease this related to the quantity and that related to update route of lineItems